package.json

```
@@ -12,8 +12,10 @@
     "standard": "echo 'standard is no longer being used, use `npm run eslint` instead!' && exit 1",
     "dev": "webpack --config webpack.dev.js --progress --colors --watch",
     "heroku-prebuild": "bin/heroku",
-    "build": "webpack --config webpack.prod.js --progress --colors --bail",
-    "start": "tsc && sequelize db:migrate && node built/lib/app.js"
+    "build": "npm run-script build-backend && npm run-script build-frontend",
+    "build-backend": "tsc",
+    "build-frontend": "webpack --config webpack.prod.js --progress --colors --bail",
+    "start": "[ ! -e built/ ] && npm run-script build-backend; sequelize db:migrate && node built/lib/app.js"
```

```suggestion
    "start": "[ -d built/ ] && npm run-script build-backend; sequelize db:migrate && node built/lib/app.js"
```
Just to be sure it is the directory.

Also I am not sure if start should call build-frontend or build-backend should be removed.

---

I think it makes sense to try to build the application if it's not already done. In best case we run successful, in worst case we die anyway.

---

Also we should not check if it's a directory. The trailing slash makes sure that a directory is available. And in worst case we'll fail to call the binary anyway.